### PR TITLE
Update GetMultipleTest.cs to use nunit

### PR DESCRIPTION
**Pull request by maxinfet**
Updated get multiple test to use nunit.

### CODE REVIEW

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 using TestStack.White.UIItems;
 using TestStack.White.UIItems.Finders;
-using Xunit;
 
 namespace TestStack.White.UITests.Scenarios
 {
-    public class GetMultipleTest : WhiteTestBase
+    [TestFixture(WindowsFramework.WinForms)]
+    [TestFixture(WindowsFramework.Wpf)]
+    public class GetMultipleTest : WhiteUITestBase
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: Should be GetMultipleTests

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 2]
```diff
@@ -23,14 +23,15 @@ void GetControlBasedOnIndex(WindowsFramework framework)
...
                     "Failed to get ControlType=edit,AutomationElementIdentifiers.NameProperty=Button,Index=4":
                     "Failed to get (ControlType=edit or ControlType=document),AutomationElementIdentifiers.NameProperty=Button,Index=4";
-                Assert.Equal(expected, exception.Message);
+                Assert.That(expected, Is.EqualTo(exception.Message));
             }
         }
 
-        void GetMultipleButtons(WindowsFramework framework)
+        [Test]
+        public void GetMultipleButtons()
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: Should be GetMultipleButtonsTest

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-        protected override void ExecuteTestRun(WindowsFramework framework)
+        public GetMultipleTest(WindowsFramework framework) : base(framework)
         {
-            RunTest(() => GetMultipleButtons(framework));
-            RunTest(() => GetControlBasedOnIndex(framework));
         }
 
-        void GetControlBasedOnIndex(WindowsFramework framework)
+        [Test]
+        public void GetControlBasedOnIndex()
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: Should be GetControlBasedOnIndexTest

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 4]
```diff
@@ -23,14 +23,15 @@ void GetControlBasedOnIndex(WindowsFramework framework)
                 Assert.NotNull(button);
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: Change to Assert.That(xxx, Is.Not.Null)

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 5]
```diff
@@ -23,14 +23,15 @@ void GetControlBasedOnIndex(WindowsFramework framework)
...
 
                 var exception = Assert.Throws<AutomationException>(() => MainWindow.Get<TextBox>(SearchCriteria.ByNativeProperty(AutomationElement.NameProperty, "Button").AndIndex(4)));
-                var expected = framework == WindowsFramework.Wpf?
+                var expected = Framework == WindowsFramework.Wpf?
                     "Failed to get ControlType=edit,AutomationElementIdentifiers.NameProperty=Button,Index=4":
                     "Failed to get (ControlType=edit or ControlType=document),AutomationElementIdentifiers.NameProperty=Button,Index=4";
-                Assert.Equal(expected, exception.Message);
+                Assert.That(expected, Is.EqualTo(exception.Message));
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: For Execption asserts, use like this:
```
Assert.That(() => { MainWindow.Get<Button>(SearchCriteria.ByAutomationId("foo")); },
	Throws.TypeOf<AutomationException>().With.
	Message.EqualTo(String.Format("Failed to get (ControlType={0} or ControlType={1}),AutomationId=foo",
		ControlType.Button.LocalizedControlType, ControlType.CheckBox.LocalizedControlType)));
```

File: src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs [hunk 6]
```diff
@@ -38,31 +39,31 @@ void GetMultipleButtons(WindowsFramework framework)
             try
             {
                 var buttons = window.GetMultiple(SearchCriteria.ByNativeProperty(AutomationElement.NameProperty, "Button")).OfType<Button>();
-                Assert.Equal(3, buttons.Count());
+                Assert.That(3, Is.EqualTo(buttons.Count()));
```
Comment (Roemer) [inline src/TestStack.White.UITests/Scenarios/GetMultipleTest.cs]: Count Asserts should look like:
```
 Assert.That(conditions, Has.Count.EqualTo(2));
```

**Comment by Roemer** [comment]
Added some feedback.

**Comment by maxinfet** [comment]
Sorry, the original change and the changes you suggest are now in pull request https://github.com/TestStack/White/pull/346

**Pull request closed without merging**